tests/correctness/fast-attr/valid/fast6inc.expect

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
I think for this test specifically, we'd like to track the cycle count.

Reply:
I was thinking about that. The issue I was running into is what if we modify the compilation process to add some other form of optimization that improves the cycle count further?

Reply:
I think that is probably fine. This is also a different kind of correctness test which requires exact cycle count information (maybe like the `static` tests) so we should put it into a different folder where all tests always report their cycle counts?

Reply:
I was also wondering if we could just do the fuzzing idea and guarantee the `\leq`? But that's not as strong. Ok, for now I'll add back in the cycle counts; of course, the very point of snapshot testing is that you can inevitably update your snapshots on significant changes.